python/GafferSceneUI/SetFilterUI.py

```diff
@@ -60,16 +60,19 @@
 			A set expression that computes a set that defines
 			the locations to be matched.
 
-			For example, the expression "mySpheresSet | myCubesSet"
+			For example, the expression `mySpheresSet | myCubesSet`
 			will create a set that contains all objects in
-			mySpheresSet and myCubesSet.
+			`mySpheresSet` and `myCubesSet`.
 
-			Gaffer supports the union operator (|) as shown in the
-			example and also provides intersection (&) and difference (-)
+			Gaffer supports the union operator (`|`) as shown in the
+			example and also provides intersection (`&`) and difference (`-`)
 			operations for set expressions. Names of locations
 			can be used to represent a set that contains only
 			that one location.
 
+			In addition, the `in` and `containing` operators can be
+			used to query descendant and ancestor matches.
```

Comment:
I think we would benefit from a little more exposition here. Maybe something like "For example, `materialA in assetB` will select all locations in the `materialA` set that are at or below locations in the `assetB` set."?

Reply:
@johnhaddon hows [this](https://github.com/GafferHQ/gaffer/pull/3866/commits/3e14e1fe99e956260bc45972009e2423c6575c77)?